dataclasses_json/core.py

```diff
@@ -201,10 +201,10 @@ def _decode_dataclass(cls, kvs, infer_missing):
                     init_kwargs[field.name] = overrides[field.name].decoder(
                         field_value
                     )
-                except:
+                except:  # noqa: E722
```

Comment:
I would suggest to do the same as above here as well (raise from the original exception)